doc/tutorials/triplet_stdp_synapse/triplet_stdp_synapse.ipynb

```diff
@@ -135,7 +135,7 @@
     "    os.makedirs(\"models/neurons\")\n",
     "if not os.path.isdir(\"models/synapses\"):\n",
     "    os.makedirs(\"models/synapses\")\n",
-    "resp = requests.get(\"https://raw.githubusercontent.com/nest/nestml/master/models/neurons/iaf_psc_delta.nestml\")\n",
+    "resp = requests.get(\"https://raw.githubusercontent.com/pnbabu/nestml/remove_end/models/neurons/iaf_psc_delta.nestml\")\n",
```

Comment:
Could you create an issue to fix these URLs after the merge?

Reply:
Created #881.